### PR TITLE
Add Returned action, Fix context IRI, sync style with Search Profile

### PR DESCRIPTION
**Pull request by mgylling**

### CODE REVIEW

File: profiles/toollaunch/caliper-profile-toollaunch-v1p1.html [hunk 1]
```diff
@@ -163,8 +171,13 @@ <h3>Tool Launch Event</h3>
         src="assets/tl-model.png" alt="Overview of Tool Launch Profile"/>
                 
         <p>The Tool Launch Profile models a Person launching a SoftwareApplication 
-          at a given time and in a given context.</p>
-        
+          at a given time and in a given context. It also provides the ability to 
+          model a "return response", which in the context of LTI [[LTI-13]] 
+          typically consists of a <a href="https://www.imsglobal.org/spec/lti/v1p3/#lis-message-variables">
+          redirect URL</a> or a specific resonse message, as defined in LTI Deep
```
Comment (ViktorHaag) [inline profiles/toollaunch/caliper-profile-toollaunch-v1p1.html]: sp -- `resonse` --> `response`

Comment (ViktorHaag) [inline profiles/toollaunch/caliper-profile-toollaunch-v1p1.html]: We should say "for example, Deep Linking" -- the pattern of launch message followed by a separate response message may well be around with other kinds of LTI things in the future, too.

File: profiles/toollaunch/local-biblio.js [hunk 2]
```diff
@@ -46,6 +46,12 @@ var _localBiblio = {
   "CALIPER-TOOLLAUNCH-11-ERRATA": {
     title: "Caliper Analytics® Tool Launch Profile 1.1 Errata",
     href: "https://www.imsglobal.org/spec/caliper-tl/v1p1/errata/",
-    publisher: "IMS Global Learning Consortium",
+    publisher: "IMS Global Learning Consortium"
+  }
+  ,        
+  "LTI-DL": {
+    title: "LTI Deep Linking @@@ FIXME",
```
Comment (ViktorHaag) [inline profiles/toollaunch/local-biblio.js]: Seems like this needs fixing before merging. 8)

**Comment by mgylling** [comment]
@ViktorHaag, @arwhyte: PR amended

**Pull request merged**